spec/classes/lightblue/application/migrator_spec.rb

```diff
@@ -40,9 +40,9 @@
         }
       )
       
-      should contain_file(client_config) \
-        .that_comes_before("Service[#{service_name}]") \
-        .that_notifies("Service[#{service_name}]")
+      should contain_lightblue__client__configure(client_config) \
```

Comment:
>* what is contain_lightblue__client__configure and where is it defined?

This is when you want to test that a class (or define as in this case) has been included by the class being tested. When '::' are used, as in lightblue::client::configure, then for testing purposes you need to replace the colons with underscores.
see: http://rspec-puppet.com/matchers/

>*if it should notify a service, what is the "comes before" bit afterwards?
You make an excellent point. A new commit to come.